### DISCUSSION TITLE
Google Drive for Desktop

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#19] [General]
# Google Drive for Desktop

Check if Google Drive for Desktop is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/424248ba-b38f-41a8-a088-66b6bcf0a1e0

**Comment by talynone** [comment]
Check out RAiDrive as an alternative https://www.raidrive.com/

**Comment by talynone** [comment]
Click on "I have the same question" button to vote for interest in a Google Drive ARM64 port: https://support.google.com/drive/thread/280877617/google-drive-for-windows-arm64?hl=en

**Comment by marcpems** [comment]
Google announced coming: https://support.google.com/drive/thread/280877617?hl=en&msgid=294908051
https://www.windowscentral.com/software-apps/google-drive-is-finally-coming-to-windows-on-arm

**Comment by talynone** [comment]
Beta download for ARM64 now available:

https://support.google.com/drive/thread/308624582/drive-for-desktop-arm64-support?hl=en&sjid=10892867771861133968-NC

**Reply by OpenSource03** [reply]
Updated, thanks
